tests/Internal/RejectedPromiseTest.php

```diff
@@ -9,14 +9,17 @@
 use React\Promise\PromiseTest\PromiseSettledTestTrait;
 use React\Promise\TestCase;
 
+/**
+ * @template T
+ */
 class RejectedPromiseTest extends TestCase
 {
     use PromiseSettledTestTrait,
         PromiseRejectedTestTrait;
 
     public function getPromiseTestAdapter(callable $canceller = null): CallbackPromiseAdapter
     {
-        /** @var ?RejectedPromise */
+        /** @var ?RejectedPromise<T> */
```

Comment:
Should probably be `?RejectedPromise<never>`, no?

Reply:
Correct.